### DISCUSSION TITLE
vue-3-form-submit-event-handling/

### DISCUSSION THREAD

**Discussion by giscus** [zxuqian/zxuqian.cn#61] [Announcements]
# vue-3-form-submit-event-handling/

这篇文章我们看一下对于表单整体的提交事件应该如何处理，并引入一个事件修饰符的概念。

https://zxuqian.cn/vue-3-form-submit-event-handling/

**Comment by xiamuguizhi** [comment]
B站过来的，支持老师

**Reply by zxuqian** [reply]
感谢🙏
